pymatgen/analysis/adsorption.py

```diff
@@ -19,11 +19,12 @@
 from pymatgen.analysis.structure_matcher import StructureMatcher
 from pymatgen.core import Molecule, Structure
 from pymatgen.core.operations import SymmOp
-from pymatgen.core.surface import generate_all_slabs
+from pymatgen.core.surface import Slab, generate_all_slabs
```

Comment:
This looks like a typo? `Slab` should be imported only `if TYPE_CHECKING`?

Reply:
right, feel free to change